dynamic-consumer/src/main/scala/nl/vroste/zio/kinesis/client/dynamicconsumer/DynamicConsumerLive.scala

```diff
@@ -236,7 +251,13 @@ private[client] class DynamicConsumerLive(
                             config.processor,
                             config.retrieval.streamTracker(
                               new SingleStreamTracker(
-                                StreamIdentifier.singleStreamInstance(streamName),
+                                streamIdentifier match {
+                                  case StreamIdentifier.StreamIdentifierByName(streamName) =>
+                                    software.amazon.kinesis.common.StreamIdentifier.singleStreamInstance(streamName)
```

Comment:
`StreamIdentifier` is already a thing in aws. 
Any good idea for a better name?

Reply:
Fine with me to use the fully qualified name from the amazon library here